### ISSUE TITLE
accessing native window properties

### ISSUE THREAD

**Issue by tbranyen** [cowboy/jquery-hashchange#9]
In blackberry 4.6 window[ str_location ][ str_href ] will result in a TypeError exception being raised.  Custom properties added to the window object can be access in the hash notation, but native properties cannot.

**Comment by cowboy** [comment]
Lark, can you test and see which of these works?

```
window['location']['href']
window['location'].href
window.location['href']
window.location.href
location['href']
location.href
```

**Comment by tbranyen** [comment]
window['location']['href'] - fails
window['location'].href - fails
window.location['href'] - works
window.location.href - works
location['href'] - works
location.href - works

**Comment by cowboy** [comment]
Changed window['location']['href'] to location.href to fix OMG FUN bug in blackberry 4.6. Closed by 400b3573ded39d4f5c10320de9ab43b08bbd2e1a.
